projects/fal/src/fal/toolkit/utils/download_utils.py

```diff
@@ -296,7 +296,7 @@ def download_model_weights(url: str, force: bool = False):
         A Path object representing the full path to the downloaded model weights.
     """
     # This is not a protected path, so the user may change stuff internally
-    weights_dir = FAL_MODEL_WEIGHTS_DIR / _hash_url(url)
+    weights_dir = Path(FAL_MODEL_WEIGHTS_DIR / _hash_url(url))
```

Comment:
if you are already using `FAL_MODEL_WEIGHTS_DIR / something` the `/` operator should be defined for `FAL_MODEL_WEIGHTS_DIR` (and it is actually a `Path` instance).

What is this actually fixing?

Reply:
`FAL_MODEL_WEIGHTS_DIR` is a `PurePath` actually which do not have functions like `exists`. You need to convert it to a `Path` to be able to call those functions. https://github.com/fal-ai/fal/blob/43380ed27779223e580be9589dc51313b583fa40/projects/fal/src/fal/toolkit/utils/download_utils.py#L15

Reply:
nice, did not know!